lib/src/repo.rs

```diff
@@ -251,6 +257,10 @@ impl ReadonlyRepo {
             .context(&submodule_store_type_path)?;
         let submodule_store = Arc::from(submodule_store);
 
+        // We dont have an initializer for this
+        SimpleWorkspaceStore::load(&repo_path)?
+            .add(&WorkspaceNameBuf::from("default"), workspace_root)?;
+
```

Comment:
Perhaps, this can be done by caller, somewhere around `init_working_copy()` or `working_copy_factory.init_working_copy()`. `Repo` shouldn't know workspace.